### PR TITLE
make app code compliant to public interface policy

### PR DESCRIPTION
**Pull request by georgehrke**
fixes https://github.com/owncloud/firstrunwizard/issues/22

code-check now passes
```
$ php occ app:check-code firstrunwizard
Analysing /var/www/apps/firstrunwizard/ajax/disable.php
 0 errors
Analysing /var/www/apps/firstrunwizard/ajax/enable.php
 0 errors
Analysing /var/www/apps/firstrunwizard/appinfo/app.php
 0 errors
Analysing /var/www/apps/firstrunwizard/appinfo/routes.php
 0 errors
Analysing /var/www/apps/firstrunwizard/lib/firstrunwizard.php
 0 errors
Analysing /var/www/apps/firstrunwizard/templates/wizard.php
 0 errors
Analysing /var/www/apps/firstrunwizard/wizard.php
 0 errors
App is compliant - awesome job!
```

please review @MorrisJobke @PVince81

### CODE REVIEW

**Comment by karlitschek** [comment]
:+1:

**Comment by PVince81** [comment]
Requires https://github.com/owncloud/core/pull/16024

**Comment by georgehrke** [comment]
removed dependency on owncloud/core#16024, please review again

File: lib/firstrunwizard.php [hunk 1]
```diff
@@ -44,13 +44,9 @@ public static function disable() {
...
 	*/
-	public static function isenabled() {
-		$conf=\OCP\CONFIG::getUserValue( \OCP\User::getUser() , 'firstrunwizard' , 'show' , 1 );
-		if($conf==1) {
-			return(true);
-		}else{
-			return(false);
-		}
+	public static function isEnabled() {
+		$conf = \OCP\CONFIG::getUserValue( \OCP\User::getUser() , 'firstrunwizard' , 'show' , 1 );
```
Comment (DeepDiver1975) [inline lib/firstrunwizard.php]: \OCP\Config please

**Comment by DeepDiver1975** [comment]
@georgehrke please get rid of the static methods and apply unit tests - thx

**Comment by jancborchardt** [comment]
@DeepDiver1975 @karlitschek can you review this again?

**Comment by MorrisJobke** [comment]
@georgehrke Please rebase.

**Comment by georgehrke** [comment]
rebased

File: .travis.yml [hunk 2]
```diff
@@ -21,18 +15,23 @@ php:
...
 
-# execute any number of scripts before the test run, custom env's are available as variables
-#before_script:
-#  - if [[ "$DB" == "pgsql" ]]; then psql -c "DROP DATABASE IF EXISTS hello_world_test;" -U postgres; fi
-#  - if [[ "$DB" == "pgsql" ]]; then psql -c "create database hello_world_test;" -U postgres; fi
-#  - if [[ "$DB" == "mysql" ]]; then mysql -e "create database IF NOT EXISTS hello_world_test;" -uroot; fi
+env:
+  matrix:
+    - DB=sqlite
+>>>>>>> remove static methods and add unit tests
```
Comment (MorrisJobke) [inline .travis.yml]: This breaks Travis

**Comment by MorrisJobke** [comment]
Tested and works :+1:

I fixed the merge conflict issues too.

**Comment by MorrisJobke** [comment]
@georgehrke asked me to take care of this ;)

**Comment by MorrisJobke** [comment]
Travis pass -> merge

**Pull request merged**